incskill_site/templates/quiz1.html

```diff
@@ -132,12 +131,34 @@ <h1>Please Pick An Option</h1>
 
 
 <style>
-  *{
+
+#continue{
+    background: rgb(43, 42, 42) !important;
```

Comment:
`!important` is bad you should not use it

Reply:
I was having trouble setting the background color of the button because I didn't realize continue was misspelled. Once I fixed that I was able to delete "!important".